### DISCUSSION TITLE
Reinstall an OS without touching the others

### DISCUSSION THREAD

**Discussion by littleworm49fr** [procount/pinn#831] [General]
Hello,

I currently have a 64GB FAT32 micro-SD card inserted in my Raspberry pi 4.
I use PINN to manage BATOCERA and RASPBERRY PI OS.
Batocera works perfectly.
I would like to install Steam Link on RaspiOS but for now it crashes the OS... I haven't found the solution yet.
Anyway, when it crashes, I reinstall RaspiOS, but it also reinstalls Batocera... Which forces me to review all the Batocera settings, while it was working perfectly.

Question: is it possible to reinstall RaspiOS without touching the existing Batocera installation?

Thank you in advance for your answers, and have a nice day!

Julien

**Comment by procount** [comment]
You should choose the **RE**-install option from the Maintenance Menu.
Please see the User Manual https://github.com/procount/pinn/blob/master/README_PINN.md
and particularly https://github.com/procount/pinn/blob/master/README_PINN.md#reinstall-individual-oses

**Comment by littleworm49fr** [comment]
Ok great, thanks a lot, and sorry for not reading the manual.
Thanks again, have a nice day,
Julien
